terraform/workspace-variables/dev_app_env.yml

```diff
@@ -0,0 +1,9 @@
+---
+RACK_ENV: production
+RAILS_ENV: development
+NODE_ENV: production
+RAILS_LOG_TO_STDOUT: true
+RAILS_MAX_THREADS: 5
+RAILS_SERVE_STATIC_FILES: true
+SERVICE_TYPE: web
+DATABASE_URL: postgres://postgres@localhost:5432
```

Comment:
Looks odd to me to have the PG port in the code

And the fact that its on the same host.  In heroku the db credentials are all in a secret